app/frontend/src/features/profile/ProfileMarkdownInput.tsx

```diff
@@ -1,57 +1,37 @@
-import { Box, Button, Grid, Link, makeStyles } from "@material-ui/core";
-import Markdown from "components/Markdown";
-import ProfileTextInput from "features/profile/ProfileTextInput";
-import React, { useState } from "react";
-
-const useStyles = makeStyles({
-  reverseAlignment: {
-    textAlign: "end",
-  },
-});
+import { Typography } from "@material-ui/core";
+import MarkdownInput from "components/MarkdownInput";
+import React from "react";
+import { Control } from "react-hook-form";
 
 interface ProfileMarkdownInputProps {
+  className?: string;
+  control: Control;
+  defaultValue?: string;
   id: string;
-  onChange: (value: string) => void;
-  value: string;
   label: string;
+  name: string;
 }
 
 export default function ProfileMarkdownInput({
+  className,
+  control,
+  defaultValue = "",
   id,
-  onChange,
-  value,
   label,
+  name,
 }: ProfileMarkdownInputProps) {
-  const classes = useStyles();
-  const [preview, setPreview] = useState(false);
   return (
-    <>
-      <Grid container spacing={2}>
-        <Grid item xs={12} md={preview ? 6 : 12}>
-          <ProfileTextInput
-            id={id}
-            label={label}
-            rowsMax={5}
-            multiline
-            fullWidth
-            onChange={(event) => onChange(event.target.value)}
-            value={value}
-          />
-          <Box className={classes.reverseAlignment}>
-            <Button component={Link} href="https://www.markdowntutorial.com/">
-              Formatting?
-            </Button>
-            <Button component={Link} onClick={() => setPreview(!preview)}>
-              Preview?
-            </Button>
-          </Box>
-        </Grid>
-        {preview && (
-          <Grid item xs={12} md={6}>
-            <Markdown source={value} />
-          </Grid>
-        )}
-      </Grid>
-    </>
+    <div className={className}>
+      <Typography variant="h2" id={id}>
+        {label}
+      </Typography>
+      <MarkdownInput
+        control={control}
+        defaultValue={defaultValue}
+        id={id}
+        labelId={`${id}-label`}
```

Comment:
```suggestion
        labelId={id}
```
The other value was flagging up as invalid in the accessibility view

Reply:
It was something else, I wasn't setting the h2 id to this value.